config/CMakeLists.txt

```diff
@@ -69,6 +70,14 @@ check_symbol_exists(FD_CLOEXEC "fcntl.h" HAVE_FD_CLOEXEC)
 if(HAVE_LANGINFO_H)
   check_symbol_exists(CODESET "langinfo.h" HAVE_NL_LANGINFO_CODESET)
 endif()
+check_c_source_compiles("
+  #include <termkey.h>
+  int main(int argc, char** argv)
+  {
+    int tk_flag = TERMKEY_FLAG_NOSTART;
+    return 0;
+  }"
+  HAVE__TERMKEY_FLAG_NOSTART)
```

Comment:
This isn't working. `check_symbol_exists(TERMKEY_FLAG_NOSTART, ...)` doesn't work either, because `TERMKEY_FLAG_NOSTART` is an enum field.